mlflow/data/spark_dataset.py

```diff
@@ -101,6 +109,13 @@ def targets(self) -> Optional[str]:
         """
         return self._targets
 
+    @property
+    def predictions(self) -> Optional[str]:
+        """
+        The name of the predictions column. May be ``None`` if no predictions column is available.
```

Comment:
The current phrasing makes me think that there is no prediction column at all, rather than it being unspecified. WDYT of:

```suggestion
        The name of the predictions column. May be ``None`` if no predictions column was specified.
```

Reply:
Great idea - done!